contracts/package.json

```diff
@@ -1,7 +1,9 @@
 {
   "name": "@dcgrants/contracts",
   "version": "0.0.1",
-  "dependencies": {},
+  "dependencies": {
+    "@openzeppelin/contracts": "^4.2.0"
```

Comment:
Do we want to fix the version as opposed to to having the `^` ? aka
`"@openzeppelin/contracts": "4.2.0"`

Reply:
I think this is ok. It's pinned in the lock file and OZ is pretty stable, especially around ERC20 behavior which is what we're using it for